common/persistence/sql/domains.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE domains(
+  id CHAR(36) UNIQUE,
+  retention INT,
+  emit_metric TINYINT, /* we are using TINYINT for boolean... should we be? */
+/* end domain_config */
+  config_version BIGINT,
+  db_version BIGINT,
+/* domain */
+  name CHAR(100) PRIMARY KEY,
```

Comment:
I suspect this is not a correct use of PRIMARY KEY

Reply:
name can be primary key. Then you need to create an index on id so you can do efficient select.